### PR TITLE
Add gridcapa cse starter and app 

### PR DESCRIPTION
**Pull request by kahyami**
**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem ?** *If so, link to this issue using `'#XXX'` and skip the rest*
No

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*

Feature

**What is the current behavior?** *(You can also link to an open issue here)*

**What is the new behavior (if this is a feature change)?**
Add a two modules: spring boot starter to be used in Spring application to use gridcapa-cse and gridcapa-app 

**Does this PR introduce a breaking change?** *(What changes might users need to make in their application due to this PR?)*
No

**Other information**:

(if any of the questions/checkboxes don't apply, please delete them entirely)

### CODE REVIEW

File: gridcapa-cse-app/src/main/java/com/farao_community/farao/gridcapa_cse/app/configurations/AmqpConfiguration.java [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+import org.springframework.amqp.rabbit.listener.SimpleMessageListenerContainer;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+/**
+ * @author Amira Kahya {@literal <amira.kahya at rte-france.com>}
+ */
+@Configuration
+public class AmqpConfiguration {
```
Comment (TheMaskedTurtle) [inline gridcapa-cse-app/src/main/java/com/farao_community/farao/gridcapa_cse/app/configurations/AmqpConfiguration.java:22]: Queue should not be an anonymous queue, so it is not defined by a queue-prefix but just a queue name. As we would have replicated services they must be plugged on same queue to be sure the message is not consumed twice.
So property "cse-request.queue" is enough.

CseRequest exchange is a fanout exchange and has no routing key. 

You can see gridcapa-task-manager I think it is more how it should look like for AMQP config.

File: gridcapa-cse-app/src/main/java/com/farao_community/farao/gridcapa_cse/app/configurations/MinioConfiguration.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    @Value("${minio.access.name}")
+    private String accessKey;
+    @Value("${minio.access.secret}")
+    private String accessSecret;
+    @Value("${minio.url}")
+    private String minioUrl;
+    @Value("${minio.bucket}")
+    private String bucket;
+    @Value("${minio.base-path}")
+    private String basePath;
```
Comment (TheMaskedTurtle) [inline gridcapa-cse-app/src/main/java/com/farao_community/farao/gridcapa_cse/app/configurations/MinioConfiguration.java:30]: cse-cc:
  -- minio:
    ---- connect:
      ------ url: ...
      ------ access-key: ...
      ------ secret-key: ...
    ---- bucket: ...
    ---- base-directory: ...

I feel like something like that would be more inline with what we did in other services

File: gridcapa-cse-app/src/main/java/com/farao_community/farao/gridcapa_cse/app/configurations/UrlWhitelistConfiguration.java [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * @author Amira Kahya {@literal <amira.kahya at rte-france.com>}
+ */
+@Configuration
+@EnableConfigurationProperties
+@ConfigurationProperties("cse-cc")
```
Comment (TheMaskedTurtle) [inline gridcapa-cse-app/src/main/java/com/farao_community/farao/gridcapa_cse/app/configurations/UrlWhitelistConfiguration.java:21]: To reduce configuration classes and to align with gridcapa projects you can use general configuration properties for all cse-cc properties. Like that but then you don't retrieve only whitelist.

File: gridcapa-cse-spring-boot-starter/src/main/java/com/farao_community/farao/gridcapa_cse/starter/CseClientProperties.java [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+            this.exchange = exchange;
+        }
+
+        public String getRoutingKey() {
+            return routingKey;
+        }
+
+        public void setRoutingKey(String routingKey) {
+            this.routingKey = routingKey;
+        }
```
Comment (TheMaskedTurtle) [inline gridcapa-cse-spring-boot-starter/src/main/java/com/farao_community/farao/gridcapa_cse/starter/CseClientProperties.java:46]: No need for routing key

File: gridcapa-cse-app/src/main/java/com/farao_community/farao/gridcapa_cse/app/configurations/UrlWhitelistConfiguration.java (hunk 3 above)
Comment (kahyami) [reply gridcapa-cse-app/src/main/java/com/farao_community/farao/gridcapa_cse/app/configurations/UrlWhitelistConfiguration.java:21]: The same here not all gridcapa projects use general configuration properties. So, we can discuss with the  team to harmonize in all projects.

File: gridcapa-cse-app/pom.xml [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+            -->
+            <dependency>
+                <groupId>org.springframework.boot</groupId>
+                <artifactId>spring-boot-dependencies</artifactId>
+                <version>${springboot.dependencies.version}</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
```
Comment (TheMaskedTurtle) [inline gridcapa-cse-app/pom.xml:78]: No need to put that it is already in farao-parent-ws

**Pull request merged**